src/dotnet-compile-fsc/AssemblyInfoOptionsCommandLine.fs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Microsoft.DotNet.Tools.Compiler
```

Comment:
that's an awesome work, and really good for dogfooding.
But speaking with sdk/vf# team, the way to go is using an `Fsc` msbuild task instead of `dotnet-compile-fsc`.
that mean for `preview3` we are not going to use `dotnet-compile-fsc` anymore.

For `preview2` is not going to be updated too (there is one updated package waiting for publish, but useless now).

so no need to replace c# version with f#, if we are going to never publish that on the wild.